api/handler.py

```diff
@@ -96,3 +100,53 @@ def getDevicesNearInfected(event, context):
     response = {"statusCode": 200, "body": json.dumps(body)}
 
     return response
+
+
+def getDeviceRisk(event, context):
+    conn = make_conn()
+    cursor = conn.cursor()
+    deviceId = event['deviceId']
```

Comment:
I think this would be `event["body"]["deviceId"]`?

Reply:
yes, it looks like I need to update all my handlers to take params from the body